src/components/fileUpload/resume.tsx

```diff
@@ -101,27 +99,39 @@ const ResumeUpload: React.FC<ResumeUploadProps> = ({ user_id, url }) => {
   };
 
   return (
-    <div className="upload-form max-w-md mx-auto rounded shadow-md p-4">
+    <div className="upload-form max-w-md mx-auto rounded-md shadow-md p-6  border-gray-200">
       <div className="space-y-6 flex flex-col items-center">
         <div className="flex items-center justify-center w-full">
           {previewUrl ? (
-            <Image
-              src={previewUrl}
-              width={160} // Updated width and height for clearer preview size
-              height={128}
-              alt="Resume Preview"
-              className="w-40 h-32 object-cover border rounded"
-            />
-          ) : (
-            <div className="flex items-center border rounded p-2 w-full justify-center">
+            selectedResume?.type === 'image/png' ? (
```

Comment:
@harshit-kharayat23 please make separate component for this and handle in if else or switch.